### DISCUSSION TITLE
how can I add a new font

### DISCUSSION THREAD

**Discussion by Kingwulin** [Bodmer/TFT_eSPI#2247] [Q&A - Fonts]
Hi,
below is the font I want to use,I just need the digit and some simple char like ":","-"
and how can I add my new font to my code
[Uploading LLPixelFun-Regular(1).zip…]()

**Comment by Bodmer** [comment]
The link does not work. Just drop the zip file in the dialogue window.

**Reply by Kingwulin** [reply]
> The link does not work. Just drop the zip file in the dialogue window.

[LLPixelFun-Regular.zip](https://github.com/Bodmer/TFT_eSPI/files/10209303/LLPixelFun-Regular.zip)

**Reply by Bodmer** [reply]
If you are using a processor with at least 64Kbytes available RAM then load this library and run the examples.
https://github.com/takkaO/OpenFontRender

The instructions for font conversion are in the header comments.

**Reply by Kingwulin** [reply]
> If you are using a processor with at least 64Kbytes available RAM then load this library and run the examples. https://github.com/takkaO/OpenFontRender
> 
> The instructions for font conversion are in the header comments.

I am using esp8266, the RAM space may be insufficient，I need all fonts, only a small part of them and fonts of specified size，
is there a tool that can convert some required fonts

**Reply by Bodmer** [reply]
In that case this example is a good starting point:
https://github.com/Bodmer/TFT_eSPI/tree/master/examples/Generic/Local_Custom_Fonts

Read the header comments in the two files and watch the YouTube tutorial.
